internal/tiltfile/tiltfile_test.go

```diff
@@ -3854,6 +3854,44 @@ local_resource("toplvl-local", "echo hello world", ["foo/baz", "foo/a.txt"])
 	}, ltTop.LocalRepos())
 }
 
+func TestLocalResourceIgnoreOnly(t *testing.T) {
+	f := newFixture(t)
+	defer f.TearDown()
+
+	f.file(".dockerignore", "**/**.c")
+	f.file("Tiltfile", "include('proj/Tiltfile')")
+	f.file("proj/Tiltfile", `
+local_resource("test", "echo hi", ["foo"], ignore=["**/*.a", "foo/bar.d"], only=["**/bar.*", "foo/baz"])
```

Comment:
can you make sure this errors if there are globs in the `only`, like `docker_build` does?

Reply:
hm, I think at that point we might as well just not have 'only'?

Reply:
ya, exactly. 

the `only` parameter was added to `docker_build`, because in `docker_build`, there's a meaningful difference between "this is the root of my docker context", and "these are the files included in my docker context". The first one is what dockerignore files are relative to.

what i'm really worried about is a situation where, by accident, "deps" and "only" become subtly different implementations of the same thing, with subtly different bugs and gotchas.